copter/source/docs/precision-landing-and-loiter.rst

```diff
@@ -0,0 +1,116 @@
+#. Activate :ref:`Land mode <land-mode>` to initiate Precision Landing.
+
+.. note::
+   Precision Landing requires a valid horizontal position estimate and a steady attitude solution. Poor GPS/EKF or a noisy rangefinder will degrade performance.
+
+
+Operation
+=========
+
+With Precision Landing enabled and Land mode activated, the external landing target measurements override the GPS solution for the final approach.
+
+If the landing target is not found, the vehicle will continue descending vertically until the height specified by :ref:`PLND_ALT_MAX <PLND_ALT_MAX>`.
+
+If the landing target is lost, the behaviour depends on :ref:`PLND_STRICT <PLND_STRICT>`:
+
+- **0** – continue landing using standard Land mode logic (not strict).
+- **1** – perform landing retries until :ref:`PLND_RET_MAX <PLND_RET_MAX>` is reached, then land normally.
+- **2** – perform landing retries until :ref:`PLND_RET_MAX <PLND_RET_MAX>` is reached, then hover instead of landing.
+
+When a landing retry is triggered, the vehicle will climb to a height/position determined by :ref:`PLND_RET_BEHAVE <PLND_RET_BEHAVE>`.
+It will then descend again in hopes of detecting the landing target.
+It will keep doing this until :ref:`PLND_RET_MAX <PLND_RET_MAX>` retries have occurred.
+Then, depending on :ref:`PLND_STRICT <PLND_STRICT>`, the vehicle will either just land, or it will stay in the air hovering
+(useful for, say, landing on boats where you don’t want to land on water).
+
+If the landing target is lost below the height specified by :ref:`PLND_ALT_MIN <PLND_ALT_MIN>`, the vehicle will continue landing vertically.
+
+ArduPilot's EKF assumes the landing target is stationary. Set :ref:`PLND_OPTIONS <PLND_OPTIONS>` bit 0 if landing on a moving target.
+
+Repositioning manually by the pilot during the landing will abort the landing unless :ref:`PLND_OPTIONS <PLND_OPTIONS>` bit 1 is set.
+
+Final landing speed may be reduced below :ref:`LAND_SPEED <LAND_SPEED>` as necessary to assure a precise touchdown.
+This can be disabled for a faster final land speed by setting :ref:`PLND_OPTIONS <PLND_OPTIONS>` bit 2.
+
+..  youtube:: plM5BJY34Bc
+    :width: 100%
+
+*This video demonstrates the behaviour of different PLND_STRICT settings.*
+
+
+Parameters
+==========
+
+- :ref:`PLND_ENABLED <PLND_ENABLED>`: Set to 1 to enable Precision Landing. Refresh parameters to see following:
+- :ref:`PLND_TYPE <PLND_TYPE>`: Sets the type of landing target position source.
+- :ref:`PLND_ORIENT <PLND_ORIENT>` / :ref:`PLND_YAW_ALIGN <PLND_YAW_ALIGN>`: Sets the mounting orientation of the landing target camera/sensor.
+- :ref:`PLND_LAND_OFS_X <PLND_LAND_OFS_X>` / :ref:`PLND_LAND_OFS_Y <PLND_LAND_OFS_Y>`: Sets the desired landing position of the camera/sensor relative to the target.
+- :ref:`PLND_CAM_POS_X <PLND_CAM_POS_X>` / :ref:`PLND_CAM_POS_Y <PLND_CAM_POS_Y>` / :ref:`PLND_CAM_POS_Z <PLND_CAM_POS_Z>`: Sets the camera/sensor position relative to the vehicle body frame.
```

Comment:
Again not a blocker but we should provide a link to the sensor offsets page which explains the "frame" of the X, Y and Z values

Reply:
done